test/framework/src/main/java/org/opensearch/bootstrap/BootstrapForTesting.java

```diff
@@ -46,9 +46,9 @@
 import org.opensearch.common.settings.Settings;
 import org.opensearch.core.common.Strings;
 import org.opensearch.core.util.FileSystemUtils;
+import org.opensearch.javaagent.bootstrap.AgentPolicy;
```

Comment:
we cannot use the [PolicyFile implementation in JDK21](https://github.com/openjdk/jdk21u/blob/master/src/java.base/share/classes/sun/security/provider/PolicyFile.java)? via ```--add-exports java.base/sun.security.provider=ALL-UNNAMED```